cla-backend-go/github_organizations/handlers.go

```diff
@@ -21,8 +24,15 @@ func Configure(api *operations.ClaAPI, service Service, eventService events.Serv
 		func(params github_organizations.GetProjectGithubOrganizationsParams, claUser *user.CLAUser) middleware.Responder {
 			reqID := utils.GetRequestID(params.XREQUESTID)
 			ctx := context.WithValue(context.Background(), utils.XREQUESTID, reqID) // nolint
+
 			result, err := service.GetGithubOrganizations(ctx, params.ProjectSFID)
 			if err != nil {
+				if strings.ContainsAny(err.Error(), "getProjectNotFound") {
```

Comment:
I would prefer that we instead of checking strings...that we create a custom error type, similar to: https://github.com/communitybridge/easycla/blob/master/cla-backend-go/utils/errors.go#L8-L20. Then we can check the type rather than comparing string values (which can easily be changed by accident).  Thoughts?

Reply:
Yeah, I have done it some other way.
Thanks